applications/DAOsign.md

```diff
@@ -0,0 +1,183 @@
+
+### Team Code Repos
+
+- https://github.com/ConsiderItDone
+- https://github.com/DAOsign
+
+Please also provide the GitHub accounts of all team members. If they contain no activity, references to projects hosted elsewhere or live are also fine.
+
+- [Eugene Fine](https://github.com/ConsiderItDone)
+- [Ramil Amerzyanov](https://github.com/ramilexe)
+- [Oleksandra Burmenska](https://github.com/Burmenska)
+- [Misha Kushka](https://github.com/kushkamisha)
+
+### Team LinkedIn Profiles (if available)
+
+- [Eugene Fine](https://www.linkedin.com/in/eugenefine/)
+- [Ramil Amerzyanov](https://www.linkedin.com/in/ramil-amerzyanov/)
+- [Oleksandra Burmenska](https://www.linkedin.com/in/oleksandra-burmenska/)
+- [Misha Kushka](https://www.linkedin.com/in/mkushka/) 
+
+
+## Development Status :open_book:
+
+We are currently working on a tech demo (kind of a pre-beta version) and non-blockchain version of DAOsign is available here: https://testnet.daosign.org
+
+## Development Roadmap :nut_and_bolt:
+
+**Overview**
+
+Total Estimated Duration: 2 months
+Full-Time Equivalent (FTE): 2 FTE per month
+Total Costs: 48,100 USD
+
+Milestone 1 - Smartcontracts. 
+Total Costs: 40,300 USD
+
+| Number | Deliverable | Specification |
+| -----: | ----------- | ------------- |
+| 0a. | License |  Apache 2.0 |
+| 0b. | Documentation | We will provide both **inline documentation** of the code and a basic **tutorial** that explains how a user can upload/generate the agreement, add signers/observers and complete the signature process with storing signature proofs either in test or mainnet, which will show how the new functionality works. |
+| 0c. | Testing and Testing Guide | Core functions will be fully covered by comprehensive unit tests to ensure functionality and robustness. In the guide, we will describe how to run these tests. |
+| 0d. | Delivery format | We will provide a website that will include the agreement creation, modification and signing functionality for users with further proofs storing on Polkadot chain.
+| 0e. | Article | We will publish an **article** that explains how does DAOsign users can create agreements and store them in Polkadot test- or mainnet. It will be published in our Medium, Linkedin, Twitter and Discord channels. Apart of that, we will regularly publish grant development status on our social media. 
+| 1. | Smart Contracts | Agreement Contract |
+| 2. | Smart Contracts | EIP712 standard implementation on Ink! reusing https://github.com/Brushfam/openbrush-contracts|
+| 3. | Smart Contracts | EIP2771 standard implementation on Ink! |
+| 4. | Smart Contracts | Delegate call using EIP2771 |
+
+
+
+
+Milestone 2 - Application integration. 
+Total Costs: 7,800 USD
+
+
+| Number | Deliverable | Specification |
+| -----: | ----------- | ------------- |
+| **0a.** | License | TBD |
+| 1. | Application Integration| Application Private Key Management |
+| 2. | Application Integration | DAOsign app integration |
```

Comment:
- These deliverables are still not well specified. Please describe what will be delivered here in terms of tools, functions, etc., so that there is no confusion about what to expect here.
- The license should be updated.
- This milestone should also include the default deliverables, at least documentation and a testing guide.

Reply:
Fixed license, added documentations and testing and updated description for deliverables

Reply:
We are planning to implement SDK using JS/Typescript. It will be open source. Also, we will integrate this SDK into our DAOsign application, which will be open source as well. DAOsign is written using ReactJS on frontend and Typescript on backend

Reply:
Thanks @ramilexe could you integrate this info into the delivery tables themselves? As this is the criteria that will be used to evaluate. Thanks!

Reply:
done